pkg/builtins/bitwise.go

```diff
@@ -0,0 +1,104 @@
+package builtins
+
+import (
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
+)
+
+const BITWISE_BUILTIN_NAME = "bitwise"
+const BITWISE_CELLS_PER_INSTANCE = 5
+const BITWISE_TOTAL_N_BITS = 251
+const BIWISE_INPUT_CELLS_PER_INSTANCE = 2
+
+type BitwiseBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func BitwiseError(err error) error {
+	return errors.Wrapf(err, "Bitwise builtin error\n")
+}
+
+func ErrFeltBiggerThanPowerOfTwo(felt lambdaworks.Felt, exponent uint) error {
+	upperBound := uint64(math.Pow(2, float64(exponent)))
+	return BitwiseError(errors.Errorf("Expected felt %d to be smaller than %d", felt, upperBound))
```

Comment:
```suggestion
	return BitwiseError(errors.Errorf("Expected felt %d to be smaller than  2**%d", felt, upperBound))
```
This looks clearer to the user imo. Also if the value is constant we can hardcode it in the error message

Reply:
100%